src/lib/variants.ts

```diff
@@ -224,10 +224,9 @@ export const withinArticleViewedSettings = (history: WeeklyArticleHistory): Filt
     },
 });
 
-export const inCorrectCohort = (userCohort: UserCohort): Filter => ({
+export const inCorrectCohort = (userCohorts: string[]): Filter => ({
```

Comment:
```suggestion
export const inCorrectCohort = (userCohorts: UserCohort[]): Filter => ({
```

Reply:
This will then need the Test interface to be updated to `userCohort: UserCohort;`